packages/pds/tests/seeds/basic.ts

```diff
@@ -107,6 +107,8 @@ export default async (
     'tests/sample-img/key-landscape-small.jpg',
     'image/jpeg',
   )
+  // must ensure ordering of replies in indexing
+  await sc.network.processAll()
```

Comment:
Can you remind me why/where the ordering of replies is significant?  I think the appview at least should be permissive of this.

Reply:
I think it's less permissive of it now that we check to ensure that reply parents are valid

basically if we don't have the parent yet, I think we mark it as invalid 🤔